R-packages/covidcast/DESCRIPTION

```diff
@@ -43,6 +43,7 @@ Imports:
   grDevices,
   httr,
   jsonlite,
+  lubridate,
```

Comment:
lubridate is only used in the tests, right? I think that means [it can go in Suggests instead of Imports](https://r-pkgs.org/description.html#dependencies)

Reply:
`ymd` is a lubricate function which I used a few times in `covidcast.R`